pronto-rubocop.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.summary = 'Pronto runner for Rubocop, ruby code analyzer'
 
   s.licenses = ['MIT']
-  s.required_ruby_version = '>= 2.3.0'
+  s.required_ruby_version = '>= 2.4.0'
```

Comment:
Works around https://github.com/rubocop-hq/rubocop/issues/9482

Reply:
This breaks the CI, you should probably exclude Ruby 2.3 [like done for pronto-haml](https://github.com/prontolabs/pronto-haml/blob/8e230f3/.github/workflows/checks.yml#L15).